sdk/eventhub/azure-eventhub/README.md

```diff
@@ -40,16 +40,6 @@ There, you can also find detailed instructions for using the Azure CLI, Azure Po
 
 Interaction with Event Hubs starts with an instance of the EventHubClient class. You need the host name, SAS/AAD credential and event hub name to instantiate the client object.
```

Comment:
"Interaction with Event Hubs starts with an instance of the EventHubConsumerClient or EventHubProducerClient class. You need either the host name, SAS/AAD credential and event hub name to instantiate the client object or a connection string that consists of them."